core/src/main/java/nl/nn/adapterframework/senders/SenderWrapperBase.java

```diff
@@ -96,7 +95,7 @@ public Message sendMessage(Message message, PipeLineSession session) throws Send
 
 	@Override
 	public String getLogPrefix() {
-		return ClassUtils.nameOf(this)+" ["+getName()+"] ";
+		return super.getLogPrefix();
 	}
```

Comment:
Dan kan dit weg toch

Reply:
dat dacht ik ook, maar super is protected